src/NuGet.Clients/NuGet.CommandLine/Commands/PackCommand.cs

```diff
@@ -119,12 +116,6 @@ public override void ExecuteCommand()
             // The directory that contains msbuild
             packArgs.MsBuildDirectory = new Lazy<string>(() => MsBuildUtility.GetMsbuildDirectory(MSBuildVersion, Console));
 
-            if (Verbose)
-            {
-                Console.WriteWarning(LocalizedResourceManager.GetString("Option_VerboseDeprecated"));
-                Verbosity = Verbosity.Detailed;
```

Comment:
Verbosity is still used.  Option_VerboseDeprecated is still used by another command so I can't remove it just yet.